ci/.gitlab-ci.yml

```diff
@@ -0,0 +1,204 @@
+image: docker:18.09.7
+
+variables:
+  DOCKER_HOST: tcp://docker:2375
+  DOCKER_DRIVER: overlay2
+  IMAGE_NAME: emcmongoose/mongoose-storage-driver-pravega
```

Comment:
change image name

Reply:
yeah, sure, CI is just copied from pravega though. so don't expect much there. we still need to figure out gitlab mirroring and whatnot